api/admin/controller.py

```diff
@@ -1002,7 +1002,7 @@ def custom_lists(self, identifier_type, identifier):
                 else:
                     list, is_new = create(self._db, CustomList, name=name, data_source=staff_data_source, library=library)
                     list.created = datetime.now()
-                entry, was_new = list.add_entry(work)
+                entry, was_new = list.add_entry(work, featured=True)
```

Comment:
I feel like this should be controlled by a feature in the GUI rather than always being true.

Reply:
That said, I don't have a preference between having this always be true and having it always be false.

Reply:
I agree, but we don't have a design for that feature and until we have it implemented I prefer always true.

Reply:
OK, it's cool.